subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -895,6 +895,26 @@ uint8_t ll_rx_get(void **node_rx, uint16_t *handle)
 
 				goto ll_rx_get_again;
 #endif /* CONFIG_BT_CTLR_ADV_PERIODIC */
+
+#if defined(CONFIG_BT_CTLR_ADV_ISO)
+			} else if (rx->type == NODE_RX_TYPE_BIG_CHM_COMPLETE) {
+				(void)memq_dequeue(memq_ll_rx.tail,
+						   &memq_ll_rx.head, NULL);
+				mem_release(link, &mem_link_rx.free);
+
+				ll_rx_link_inc_quota(1);
+
+				/* Update Channel Map in BIGInfo present in
+				 * Periodic Advertising PDU.
+				 */
+				ull_adv_iso_chm_complete(rx);
+
+				mem_release(rx, &mem_pdu_rx.free);
+
+				rx_alloc(1);
```

Comment:
If both of these `1` refer to the same semantic value (i.e. number of PDUs to allocate or whatever, we could consider storing that value in a `const uint8_t` value to make it easier to understand that this number represent the same thing (if they do), and make it easier to refactor later if we would ever need to do that.